@navikt/core/react/src/alert/base-alert/close/BaseAlertClose.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { forwardRef } from "react";
+import { XMarkIcon } from "@navikt/aksel-icons";
+import { Button } from "../../../button";
+import { useRenameCSS } from "../../../theme/Theme";
+import { useI18n } from "../../../util/i18n/i18n.hooks";
+
+type BaseAlertCloseProps = React.ButtonHTMLAttributes<HTMLButtonElement>;
+
+/**
+ * @see 🏷️ {@link BaseAlertCloseProps}
+ * @example
+ * ```jsx
+ *  <BaseAlert>
+ *    <BaseAlert.Header>
+ *      <BaseAlert.Title>Info title</BaseAlert.Title>
+ *      <BaseAlert.Close onClick={...} />
+ *    </BaseAlert.Header>
+ *  </BaseAlert>
+ * ```
+ */
+const BaseAlertClose = forwardRef<HTMLButtonElement, BaseAlertCloseProps>(
+  ({ className, ...restProps }: BaseAlertCloseProps, forwardedRef) => {
+    const { cn } = useRenameCSS();
+    const translate = useI18n("global");
+
+    return (
+      <Button
+        ref={forwardedRef}
+        type="button"
+        {...restProps}
+        data-color="neutral"
+        variant="tertiary-neutral"
+        className={cn(className, "navds-base-alert__close-button")}
+        size="small"
+        aria-label={translate("close")}
```

Comment:
Consider using `title` instead to get tooltip.

```suggestion
        title={translate("close")}
```